CHANGELOG.md

```diff
@@ -14,6 +16,9 @@ and this library adheres to Rust's notion of
   new `wnaf-memuse` feature flag, to enable the heap usage of these types to be
   measured at runtime.
 
+### Changed
+- Removed temporary allocations from `Wnaf` internals for improved performance.
```

Comment:
Judging by this, you’ve gone over the commits since the last release. I verified these changes/additions, but can’t rule out that there is something else in the commits to call out here.